services/QuillLMS/client/app/bundles/Diagnostic/styles/components/_student-question.scss

```diff
@@ -141,6 +141,7 @@
   font-family: $family-monospace;
   margin: 1em 0;
   font-size: 20px;
+  width: 100%;
```

Comment:
This change was not strictly needed to fix the bug as reported but it has the same selector as the one that I had to change. 
But, this is in the diagnostic styles, and the diagnostic does not seem to have hints. Therefore, maybe we can actually just delete this selector from this CSS?

Reply:
I believe you're correct that the diagnostic will never show hints (because there are no revisions). Connect and Diagnostic used to be one app, and when they were split the code was duplicated (deduplication and recombining the various apps into one is an ongoing topic of conversation), which accounts for the unused CSS here. It should be safe to delete this selector, since it's a) not being used and b) styles in the Shared bundle will apply to all apps, so even if it were to be used it would get the CSS from there, but I would defer to your comfort level -- my inclination is that, on a bug fix, it is always nice but never necessary to do clean-up you come across along the way.